File: DataStructures/ExamPreparation-01/Problem-2-PitFortress/C#-Skeleton/PitFortressSkeleton/Classes/Minion.cs
namespace Classes
{
    using Interfaces;

    public class Minion : IMinion
    {
        public int CompareTo(Minion other)
        {
            throw new System.NotImplementedException();
        }

        public int Id { get; private set; }

        public int XCoordinate { get; private set; }

        public int Health { get; set; }
    }
}

File: DataStructures/ExamPreparation-01/Problem-2-PitFortress/Java-Skeleton/PitFortress/src/main/models/Minion.java
package main.models;

import main.interfaces.IMinion;

public class Minion implements IMinion {

    private int id;
    private int x;
    private int health;


    @Override
    public int getId() {
        return id;
    }

    @Override
    public int getX() {
        return x;
    }

    @Override
    public int getHealth() {
        return health;
    }

    public void setHealth(int value) {
        this.health = value;
    }

    @Override
    public int compareTo(Minion o) {
        return 0;
    }
}
